runtime/src/main/java/org/corfudb/protocols/service/CorfuProtocolMessage.java

```diff
@@ -45,6 +42,18 @@ private CorfuProtocolMessage() {}
         IGNORE
     }
 
+    /**
+     * Returns a ProtocolVersionMsg with default capability_vector and static
+     * corfu_source_code_version.
```

Comment:
As we are not setting any default value, change `default capability_vector` to `empty capability_vector list` maybe?